rpcs3/rpcs3qt/cheat_manager.cpp

```diff
@@ -913,11 +927,12 @@ void cheat_manager_dialog::do_the_search()
 
 void cheat_manager_dialog::update_cheat_list()
 {
-	u32 num_rows = 0;
+	size_t num_rows = 0;
 	for (const auto& name : g_cheat.cheats)
 		num_rows += g_cheat.cheats[name.first].size();
 
-	tbl_cheats->setRowCount(num_rows);
+	ASSERT(num_rows <= INT32_MAX);
+	tbl_cheats->setRowCount(static_cast<int>(num_rows));
```

Comment:
this does both the caet and the assert.
```suggestion
	tbl_cheats->setRowCount(::narrow<int>(num_rows));
```